### PR TITLE
WWT-5194: [BladeSDK] Broken mnemonic fix - Identify broken account + Emergency transfer

### PR DESCRIPTION
**Pull request by Dubiy**

### CODE REVIEW

File: src/BladeSDK.ts [hunk 1]
```diff
@@ -2233,6 +2232,80 @@ export class BladeSDK {
...
+     * 1. Create new account
+     * 2. Associate all tokens with new account that you want to transfer
+     * 3. Call this method to transfer funds to new account
+     * 4. Send some HBAR to broken mnemonic account to cover fees if needed
+     *
+     * @param seedPhrase mnemonic from account
+     * @param accountId account id (broken)
+     * @param receiverId new account id
+     * @param hbarAmount amount of HBAR to resque. Can be 0
+     * @param tokenList list of token ids to transfer all tokens. Can be empty
```
Comment (Volind) [inline src/BladeSDK.ts]: There is a limitation for number of transfers in total for one transaction.

File: src/BladeSDK.ts [hunk 2]
```diff
@@ -2233,6 +2232,80 @@ export class BladeSDK {
...
+                    if (tokenBalance) {
+                        tx.addTokenTransfer(tokenId, accountId, -1 * tokenBalance.balance)
+                        tx.addTokenTransfer(tokenId, receiverId, tokenBalance.balance)
+                    }
+                }
+            }
+
+            return tx.freezeWith(client)
+                .execute(client)
+                // .then((tx) => tx.execute(client))
```
Comment (Volind) [inline src/BladeSDK.ts]: Remove commented code.

File: src/BladeSDK.ts [hunk 3]
```diff
@@ -2233,6 +2232,80 @@ export class BladeSDK {
...
+     * @param receiverId new account id
+     * @param hbarAmount amount of HBAR to resque. Can be 0
+     * @param tokenList list of token ids to transfer all tokens. Can be empty
+     * @param checkOnly if true, will only check if mnemonic is broken. No transfer will be made
+     * @param completionKey optional field bridge between mobile webViews and native apps
+     * @returns {EmergencyTransferData}
+     * @example
+     * const receipt = await bladeSdk.brokenMnemonicEmergencyTransfer(brokenSeed, resqueAccountId, newAccountId, "0.5", ["0.0.1337"], false);
+     */
+    async brokenMnemonicEmergencyTransfer(seedPhrase: string, accountId: string, receiverId: string, hbarAmount: string, tokenList: string[], checkOnly: boolean, completionKey?: string): Promise<EmergencyTransferData> {
```
Comment (Volind) [inline src/BladeSDK.ts:2255]: Are we sure that we want to combine check and fix methods into one?

Comment (Dubiy) [reply src/BladeSDK.ts:2255]: yes, KC firstly going to collect some statistics, and after that use it (or not).
there is no need to create separate methods for that temporary feature

**Review by Volind — APPROVED** [review]

**Pull request merged**